index.html

```diff
@@ -0,0 +1,43 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Portfolio Website</title>
+    <link rel="stylesheet" href="style.css" />
+    <link
+      rel="stylesheet"
+      href="https://use.fontawesome.com/releases/v5.13.0/css/all.css"
+    />
+  </head>
+  <body>
+    <header>
+      <nav class="menu-bar">
+        <a class="menu-icon" href="">Welcome</a>
+        <a class="menu-icon1" href=""><i class="fas fa-bars"></i></a>
+      </nav>
+    </header>
+    <section>
+      <div class="container">
+        <div class="heading-container">
+          <h1 class="heading">Hey there. I’m Mariane</h1>
+          <h2 class="heading-2">I'm a software developer</h2>
+        </div>
+        <p class="text">
+          I can help you build a product , feature or website Look through some
+          of my work and experience! If you like what you see and have a project
+          you need coded, don’t hestiate to contact me.
+        </p>
+        <ul class="icons">
+          <li><i class="fab fa-github"></i></li>
+          <li><i class="fab fa-linkedin-in"></i></li>
+          <li><i class="fab fa-angellist"></i></li>
+          <li><i class="fab fa-twitter"></i></li>
+          <li><i class="fab fa-medium"></i></li>
+        </ul>
+        <img class="button-down" src="images/button.png" alt="" />
```

Comment:
## OPTIONAL
- [ ] Kindly add text to all alt tag, this helps your website more accessible to users who use screen readers, and text is always displayed in case of network downtime.